src/python/CRABClient/Commands/status.py

```diff
@@ -566,6 +566,9 @@ def terminate(states, jobStatus, target='no output'):
         # And if the dictionary is not empty, print it
         for jobtype, currStates in toPrint:
             if currStates:
+                automaticSplittFAQ = 'https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3FAQ#What_is_the_Automatic_splitting'
+                if any('-' in k for k in statusCacheInfo):
```

Comment:
@belforte, this 'any' is used already twice in the code.
Should we rather define a flag in the __call__ and pass it to the different functions?